src/Goerli-DssSpell.sol

```diff
@@ -47,18 +57,19 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
 
-    // Math
-    uint256 internal constant WAD = 10 ** 18;
-
     function officeHours() public override returns (bool) {
         return false;
     }
 
     function actions() public override {
+
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        // onboardNewCollaterals();
-        // offboardCollaterals();
+        onboardNewCollaterals();
+
+        // lock RWA007 Token in the URN
+        GemLike(RWA007).approve(RWA007_A_URN, 1 * WAD);
+        RwaUrnLike(RWA007_A_URN).lock(1 * WAD);
```

Comment:
I would move RWA related actions at the end together with the chainlog bump

Reply:
Done